### ISSUE TITLE
Server.xml for servicedesk

### ISSUE THREAD

**Issue by GrayPockets** [voxpupuli/puppet-jira#295]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: Puppet: 2018.1.4
- Distribution: Ubuntu 16.04
- Module version: 3.5.2

## How to reproduce (e.g Puppet code you use)
Install jira with product 'servicedesk' and version '3.15.1'

## What are you seeing
https://confluence.atlassian.com/jirakb/cant-start-jira-after-upgrade-to-jira-7-790629818.html

## What behaviour did you expect instead
Jira installs/starts as expected

## Output log
Server.xml has <Listener className="org.apache.catalina.core.JasperListener"/>

## Any additional information you'd like to impart
This is the part that doesn't seem to work:

```
<%- if scope.function_versioncmp([@version, '6.4.14']) <= 0 && @product =~ /^jira/ -%>
    <Listener className="org.apache.catalina.core.JasperListener"/>
<% else -%>
    <Listener className="org.apache.catalina.startup.VersionLoggerListener" />
<%- end -%>
```

**Comment by Ben-John** [comment]
We are also running into this issue. We are using product = 'servicedesk' and version = '4.1.0'. Yet the JasperListener class is still being inserted into server.xml

**Comment by GrayPockets** [comment]
This was fixed by #288, which looks like it was included in version 4.0.0

**Comment by GrayPockets** [comment]
This was a duplicate of #259
